misc/sampleconfigs/phoniebox-startup-sound.service.stretch-default.sample

```diff
@@ -1,14 +1,16 @@
 [Unit]
 Description=Phoniebox Startup Sound
-After=alsa-state.service
+After=mpd.service
```

Comment:
What does this mean if people use the spotify version using mopidy?

Reply:
Because the playout is done with mpg123, waiting for also should be enough, shouldn't it?

Reply:
after mpd.service is probably a  a problem if one uses mopdiy
i don't know what you want to express with the second command

Reply:
I'll play with it and see what I will do about this :)